modules/data-entry/src/main/frontend/src/questionnaireEditor/ObjectInput.jsx

```diff
@@ -36,15 +36,16 @@ import QuestionComponentManager from "../questionnaireEditor/QuestionComponentMa
 
 let ObjectInput = (props) => {
   let { objectKey, value, data } = props;
-  let [ selectedValue, setSelectedValue ] = useState(data[objectKey] || '');
+  const defaultValue = data[objectKey] || (objectKey === "minAnswers" ? 0 : '');
```

Comment:
I'm not a big fan of hardcoding `minAnswers` here. How does everyone feel about auto-selecting the first value in the dropdown by default, would there be any unwanted side effects? @sdumitriu @veronikaslc 
```suggestion
  const defaultValue = data[objectKey] || (Object.keys(value || {})[0] || '');
```
If the above change is made, it will of course affect all nested definitions such as `displayMode` and `dataType`.  We should also reorder keys in Question.json so that the default would be more common in practice, e.g:
```
     "displayMode": {
       "input": {},
       "list": {...},
       "list+input" : {...}
      "textbox": {},
      "hidden": {},
      "none": {}
     }
```
as well as putting `text` at the top of the `dataType` list.

Reply:
Addressed in 696870a